osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -84,6 +87,8 @@ public HUDOverlay(ScoreProcessor scoreProcessor, DrawableRuleset drawableRuleset
                         HealthDisplay = CreateHealthDisplay(),
                         Progress = CreateProgress(),
                         ModDisplay = CreateModsContainer(),
+                        LeftHitErrorDisplay = CreateHitErrorDisplay(false),
```

Comment:
`HUDOverlay` should not be making two of these. What if the user or skin decides only one is needed?

I think this PR should include the ability to choose where the meter goes (Left, Right, Both) as options in `ScoreMeterType`, as it will help you get the structure more correct.

Reply:
and add a bottom side as well

Reply:
No. I explicitly excluded bottom because I would prefer that in its own PR. The layout logic is not so straightforward as there are other moving HUD elements.

Reply:
Do we want to rebuild all the displays based on user choice?
I mean, the simple way to do that is just let left and right displays be always there, and just fade out one of them (or none)

Reply:
they should only be created when necessary.